src/services/libraryProfile/repository.ts

```diff
@@ -0,0 +1,65 @@
+import { v4 } from 'uuid';
+
+import { Member } from '@graasp/sdk';
+
+import { AppDataSource } from '../../plugins/datasource';
+import { ItemNotFound } from '../../utils/errors';
+import { IMemberProfile } from './controller';
+import { MemberProfile } from './entities/profile';
+
+const MemberProfileRepository = AppDataSource.getRepository(MemberProfile).extend({
+  async createOne(
+    args: IMemberProfile & {
+      member: Member;
+    },
+  ) {
+    const { bio, visibility = false, facebookLink, linkedinLink, twitterLink, member } = args;
+
+    const id = v4();
+
+    const memberProfile = await this.create({
+      id,
+      bio,
+      visibility,
+      facebookLink,
+      linkedinLink,
+      twitterLink,
+      member,
+    });
+    await this.insert(memberProfile);
```

Comment:
We should have a check that facebook/linkedin/twitter links are REALLY from facebook, etc . Same for patch. 
Or we could just keep the `id` of the url and build up the links ourselves. That might be more secure for us that way. Con is that it may break often if facebook, etc change their link routes (but actually it's the same if we keep the full link). What do you think?